chaoslib/litmus/spring-boot-chaos/lib/spring-boot-chaos.go

```diff
@@ -0,0 +1,329 @@
+func SetTargetPodList(experimentsDetails *experimentTypes.ExperimentDetails, clients clients.ClientSets, chaosDetails *types.ChaosDetails) error {
+	// Get the target pod details for the chaos execution
+	// if the target pod is not defined it will derive the random target pod list using pod affected percentage
+	var err error
+
+	if experimentsDetails.TargetPods == "" && chaosDetails.AppDetail.Label == "" {
+		return errors.Errorf("please provide one of the appLabel or TARGET_PODS")
+	}
+
+	experimentsDetails.TargetPodList, err = common.GetPodList(experimentsDetails.TargetPods, experimentsDetails.PodsAffectedPerc, clients, chaosDetails)
+	if err != nil {
+		return err
+	}
+
+	return nil
+
+}
+
+// PrepareChaos contains the preparation steps before chaos injection
+func PrepareChaos(experimentsDetails *experimentTypes.ExperimentDetails, clients clients.ClientSets, resultDetails *types.ResultDetails, eventsDetails *types.EventDetails, chaosDetails *types.ChaosDetails) error {
+
+	// Waiting for the ramp time before chaos injection
+	if experimentsDetails.RampTime != 0 {
+		log.Infof("[Ramp]: Waiting for the %vs ramp time before injecting chaos", experimentsDetails.RampTime)
+		common.WaitForDuration(experimentsDetails.RampTime)
+	}
+
+	switch strings.ToLower(experimentsDetails.Sequence) {
+	case "serial":
+		if err := injectChaosInSerialMode(experimentsDetails, clients, chaosDetails, eventsDetails, resultDetails); err != nil {
+			return err
+		}
+	case "parallel":
+		if err := injectChaosInParallelMode(experimentsDetails, clients, chaosDetails, eventsDetails, resultDetails); err != nil {
+			return err
+		}
+	default:
+		return errors.Errorf("%v sequence is not supported", experimentsDetails.Sequence)
+	}
+
+	// Waiting for the ramp time after chaos injection
+	if experimentsDetails.RampTime != 0 {
+		log.Infof("[Ramp]: Waiting for the %vs ramp time after injecting chaos", experimentsDetails.RampTime)
+		common.WaitForDuration(experimentsDetails.RampTime)
+	}
+	return nil
+}
+
+// CheckChaosMonkey verifies if chaos monkey for spring boot is available in the selected pod
+func CheckChaosMonkey(chaosMonkeyPort string, chaosMonkeyPath string, targetPods corev1.PodList) (bool, error) {
+	// Deleting the application pod
+	for _, pod := range targetPods.Items {
+		log.Infof("[Check]: Checking pod: %v", pod.Name)
+		resp, err := http.Get("http://" + pod.Status.PodIP + ":" + chaosMonkeyPort + chaosMonkeyPath) //nolint:bodyclose
+		if err != nil {
+			return false, err
+		}
+
+		if resp.StatusCode != 200 {
+			return false, errors.Errorf("failed to get chaos monkey endpoint on pod %v (status: %v)", pod.Name, resp.StatusCode)
+		}
+	}
+
+	return true, nil
+}
+
+// enableChaosMonkey enables chaos monkey on selected pods
+func enableChaosMonkey(chaosMonkeyPort string, chaosMonkeyPath string, pod corev1.Pod) error {
+	log.Infof("[Check]: Enabling Chaos Monkey on pod: %v", pod.Name)
```

Comment:
```suggestion
	log.Infof("[Chaos]: Enabling Chaos Monkey on pod: %v", pod.Name)
```
Since this step is a part of the chaos injection process, can we make these changes at other places as well?